app/lib/features/events/pages/event_page.dart

```diff
@@ -125,40 +125,66 @@ class CalendarEventPage extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final event = ref.watch(calendarEventProvider(calendarId));
-
+    final myRsvpStatus = ref.watch(myRsvpStatusProvider(calendarId));
+    final List<bool> rsvp = [false, false, false];
+    myRsvpStatus.maybeWhen(
+      data: (status) {
+        if (status == 'Yes') {
+          return rsvp.setAll(0, [false, false, true]);
+        }
+        if (status == 'Maybe') {
+          return rsvp.setAll(0, [false, true, false]);
+        }
+        if (status == 'No') {
+          return rsvp.setAll(0, [true, false, false]);
+        }
+      },
+      orElse: () => null,
+    );
```

Comment:
This is a weird way of representing it. I'd expect there to be an enum and a UI mapping pattern, as the [toggle button dart api example](https://api.flutter.dev/flutter/material/ToggleButtons-class.html) does it.
```dart
enum RSVP { Yes, Maybe, No, Pending };
```

Having a three-item list of bool where only one can be true is a pretty odd way of doing that...